db/migrate/20230718142009_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] According to the ERD:

![image](https://github.com/Elhambasir/blog-app/assets/77069011/2b991386-4de9-42c1-91d0-3235e28a9185)

This reference to the Users table should be under the name of `AuthorId`, to achieve this, you could do something like this:

```rb
# this will create an author_id column and a corresponding index
add_reference :comments, :author, foreign_key: { to_table: 'users' } 
```

Please consider applying this! 🤓 🔧